packages/gatsby/src/utils/dev-ssr/parse-error.ts

```diff
@@ -4,7 +4,11 @@ const sysPath = require(`path`)
 const fs = require(`fs-extra`)
 const { slash } = require(`gatsby-core-utils`)
 
```

Comment:
The TODO here

```
// TODO: Do not use source-map-support and migrate to a package that doesn't use source-map under the hood
```

can be solved with: https://github.com/cspotcode/node-source-map-support

Reply:
I removed the usage of ``require(`source-map-support`).install()`` and decided not to use alternative that also do some "magic installation" and instead explicitely use `createErrorFromString` we already have that converts stack trace to use source call sites in https://github.com/gatsbyjs/gatsby/pull/35685/commits/a1b033ae64a125cfbf12d09ba82972e4c75a4781